app/jobs/scheduled/clean_up_tags.rb

```diff
@@ -8,7 +8,14 @@ class CleanUpTags < ::Jobs::Scheduled
 
     def execute(args)
       return unless SiteSetting.automatically_clean_unused_tags
-      Tag.unused.where("tags.created_at < ?", GRACE_PERIOD_MINUTES.minutes.ago).destroy_all
+      Tag.transaction do
+        tags = Tag.unused.where("tags.created_at < ?", GRACE_PERIOD_MINUTES.minutes.ago)
+        StaffActionLogger.new(Discourse.system_user).log_custom(
+          "deleted_unused_tags",
+          tags: tags.pluck(:name),
```

Comment:
```suggestion
          tags: tags.pluck(:name).join(", ")
```

I think just add this to avoid the ["1", "2"] display in the logs.